calblueprint/ros-student
# Added ability to create self paced codes

- Added self paced/instructor led option to GenerateCodeCsvModal
- Reordered audio label for EditComponent
- Added AudioComponent for FormComponent and VideoComponent
- Add self-paced attribute to codes 
- Update onEnd methods for audio
- Make API call when component is completed instead of when next is pressed

Testing/Concerns: 
- Try creating self paced course and see if you need to finish audio files to continue
- Try creating instructor led course and see if no audio files appear and can continue w/o them
- Try going through series of slides in instructor led course (where slides aren't the same image) 
- Make API call when component is completed instead of when next is pressed --> could break something? Test by going through CoursePage and see if sidebar seems off
- Maybe should display current file name in EditComponent
